community/modules/scheduler/htcondor-base/variables.tf

```diff
@@ -69,9 +69,3 @@ variable "execute_point_roles" {
     "roles/storage.objectViewer",
   ]
 }
-
-variable "central_manager_high_availability" {
```

Comment:
Deprecate instead of delete. Message should mention that using module gives HA by default.

Reply:
I'd prefer to avoid variable-by-variable deprecations, given that:

- the HTCondor modules are marked experimental
- the htcondor-base module only exists on this branch and is replacing a broader htcondor-configure module on main

I think the most appropriate deprecation-y process here would be to adopt the `checkMovedModules` mechanism with (potentially) expanded guidance beyond "it's the same module with a new name."

Reply:
Yes to adding to checkMovedModules.

I will knock var deprecation down to nit but would still advocate for it. Your call.

User flow with old blueprint would be like (var deleted):
- create error: moved module
  - user updates blueprint
- create error: variable does not exist
  - user is confused on next steps??? maybe

User flow with old blueprint would be like (var deprecated):
- create error: moved module
  - user updates blueprint
- deploy error: fyi var has been deprecated, update your blueprint
  - user updates blueprint

Reply:
Added to checkMovedModules with guidance to read README.md in new location. README.md modified to tell user to read the example and modify blueprint.